tests/test_fetch_orderbook.py

```diff
@@ -0,0 +1,45 @@
+import json
+import unittest
+
+import pandas as pd
+
+from src.fetch.orderbook import OrderbookFetcher
+from src.models.block_range import BlockRange
+
+
+class MyTestCase(unittest.TestCase):
```

Comment:
In my opinion we should split these tests into two separate categories: `unit` and `integration`. What's your take on that?

Reply:
Don't worry I will fix this!